projects/core/src/product/services/search-page-meta.resolver.spec.ts

```diff
@@ -14,6 +15,16 @@ const mockSearchPage: Page = {
   slots: {},
 };
 
+const mockRoute: RouterState = {
+  state: {
+    params: {
+      query: 'Canon',
+    },
+  },
+} as any;
+
+const mockRoute$ = new BehaviorSubject<RouterState>(mockRoute);
```

Comment:
If mockRoute is not an observable, shall it have the $ in the name?

Reply:
fixed naming convention